src/components/sbb-navigation-marker/sbb-navigation-marker.tsx

```diff
@@ -90,9 +90,9 @@ export class SbbNavigationMarker implements ComponentInterface {
 
   // Create an array with only the sbb-navigation-action children.
   private _readActions(): void {
-    this._actions = Array.from(this._element.children).filter(
-      (e): e is HTMLSbbNavigationActionElement => e.tagName === 'SBB-NAVIGATION-ACTION'
-    );
+    this._actions = Array.from(
+      this._element.querySelectorAll('sbb-navigation-action')
```

Comment:
This is working?
Just making sure. 😄

Reply:
I thought I reverted this change. thanks for catching :)